components/SubmissionComments.tsx

```diff
@@ -53,16 +128,54 @@ export const SubmissionComments: React.FC<{
               inline
             />
             {id === c.authorId ? (
-              <div>
-                <Button
-                  outline
-                  type="danger"
-                  color="danger"
-                  onClick={() => handleDeleteComment(c)}
-                >
-                  <TrashIcon />
-                </Button>
-              </div>
+              editing === i ? (
+                <div>
+                  <Button
+                    type="light"
+                    color="black"
+                    onClick={() => {
+                      handleDiscardChange()
+                    }}
+                    data-testid="discard-button"
+                  >
+                    Discard Changes
+                  </Button>
+                  <span> </span>
+                  <Button
+                    type="info"
+                    color="white"
+                    onClick={() => {
+                      handleUpdateComment(c, currInput)
+                    }}
+                    data-testid="save-button"
+                  >
+                    Save Changes
+                  </Button>
+                </div>
+              ) : (
+                <div>
+                  <Button
+                    type="info"
+                    size="sm"
+                    color="white"
+                    onClick={() => {
+                      handleStartEditing(i, c.content)
+                    }}
+                    data-testid="edit-button"
+                  >
+                    Edit
```

Comment:
To keep the UI consistent, it's better to use an icon instead of a text like the delete comment trash icon

<img width="109" alt="image" src="https://user-images.githubusercontent.com/35906419/189523314-e1a858e4-1141-4905-aab7-5ca426bedd5a.png">

Reply:
Extra margin between the two.